meshtastic/ble_interface.py

```diff
@@ -82,6 +90,27 @@ def from_num_handler(self, _, b):  # pylint: disable=C0116
         self.should_read = True
 
     async def log_radio_handler(self, _, b):  # pylint: disable=C0116
+        log_record = mesh_pb2.LogRecord()
+        try:
+            log_record.ParseFromString(bytes(b))
+            log_record.message = log_record.message.replace("\n", "")
```

Comment:
What about instead of this we not include the new line on the sending side?  Looks great otherwise.